src/components/SectionWrapper/SectionWrapper.component.tsx

```
@@ -0,0 +1,59 @@
+    infoTooltip,
+    children,
+    gap = 4,
+}: SectionWrapperProps) => (
+    <Paper
+        component="section"
+        aria-label={`${title} section`}
+        elevation={1}
+        sx={{ p: 4, borderRadius: 4 }}
+    >
+        <Stack gap={gap}>
+            <header aria-label={`Header for ${title} section`}>
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-label' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)